tests/template_tests/filter_tests/test_join.py

```diff
@@ -54,6 +54,24 @@ def test_join08(self):
             "join08", {"a": ["Alpha", "Beta & me"], "var": mark_safe(" & ")}
         )
         self.assertEqual(output, "alpha & beta &amp; me")
+    
+    @setup({"join09": '{{ some_list|join:some_var }}'})
+    def test_join09(self):
+        some_list = ["<p>Hello World!</p>", "beta & me", "<script>Hi!</script>"]
+        some_var = "<br/>"
+        context = {"some_list": some_list, "some_var": some_var}
+        output = self.engine.render_to_string("join09", context)
+        result = '&lt;p&gt;Hello World!&lt;/p&gt;&lt;br/&gt;beta &amp; me&lt;br/&gt;&lt;script&gt;Hi!&lt;/script&gt;'
+        self.assertEqual(output, result)
```

Comment:
This is redundant with `test_join05`, I think we can remove it.

Reply:
OK, I will remove it.